autogluon/utils/tabular/ml/constants.py

```diff
@@ -10,3 +10,4 @@
 
 REFIT_FULL_NAME = 'refit_single_full'  # stack-name used for refit_single_full (aka "compressed") models
 REFIT_FULL_SUFFIX = "_FULL"  # suffix appended to model name for refit_single_full (aka "compressed") models
+REFIT_FULL_PREFIX = "FULL_"
```

Comment:
why is this prefix sometimes needed? (ie. why is suffix insufficient?)

Reply:
Removed, this was a minor hack that wasn't necessary.